content/en/docs/advanced-tasks/adding-an-admission-webhook.md

```diff
@@ -70,3 +70,111 @@ Additionally, in an attempt to prevent operator from configuring the cluster int
 - Intercept requests that target all groups
 - Intercept requests that target the `operators.coreos.com` group
 - Intercept requests that target the `ValidatingWebhookConfigurations` or `MutatingWebhookConfigurations` resources
+
+### Conversion Webhook support
+
+OLM allows Operators to ship with [CRD Conversion](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/#webhook-conversion) Webhooks when they only support the `AllNamespaces` installMode.
+Operator Authors may specify which of their owned CRDs rely on the Conversion Webhook by listing each of the CRD names in the `WebhookDefinitions.conversionCRDs` array within the operators `ClusterServiceVersion`.
+
+Please refer to the implementation of the [custom resource conversion webhook server](https://book.kubebuilder.io/multiversion-tutorial/conversion.html) as an example on how to write a conversion webhook server. 
+
+#### Example:
+
+```yaml
+...
+...
+...
+spec:
+  webhookdefinitions:
```

Comment:
```suggestion
  webhookDefinitions:
```

Reply:
I think it is all lower case: [`json:"webhookdefinitions,omitempty"`](https://github.com/operator-framework/operator-lifecycle-manager/blob/8b6a01de311f2d5ef3214110952b381ed9682fae/vendor/github.com/operator-framework/api/pkg/operators/v1alpha1/clusterserviceversion_types.go#L267)

Reply:
You're correct. I guess this is to align with `customresourcedefinitions`.